test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -14131,12 +14131,12 @@ def _unix_initd(self, hostname, op, conf_file, init_script, daemon):
             os.close(fd)
             self.du.set_pbs_config(hostname, fin=conf_file, fout=fn,
                                    confs=dconf)
-            init_cmd = ['PBS_CONF_FILE=' + fn]
+            init_cmd = ['sudo', 'PBS_CONF_FILE=' + fn]
             _as = True
         else:
             fn = None
             if (conf_file is not None) and (conf_file != self.dflt_conf_file):
-                init_cmd = ['PBS_CONF_FILE=' + conf_file]
+                init_cmd = ['sudo', 'PBS_CONF_FILE=' + conf_file]
                 _as = True
             else:
                 init_cmd = []
```

Comment:
Hey @kjakkali Thanks for updating code based on Travis failure... I think we need to add 'sudo' here also..
Actually I would suggest to initialize init_cmd = [] @ line 14114 and instead of added 'sudo' to every place we can do init_cmd += (instead init_cmd = ) and remove this line. What you say?